docs/README.md

```diff
@@ -5,33 +5,39 @@
 </p>
 
 # Monster Hunter Now Events
+## Table Of Contents
+* Calendar Subscription (Purpose)
+  - Steps to Subscribe Event Calendar <br>
+* Local Implementation of Calendar Subscription <br>
+* Tools <br>
+* References 
 
-This is a calendar subscription file for events in [Monster Hunter Now](https://monsterhunternow.com/).
+## Calendar Subscription
+This is a calendar subscription page for events in [Monster Hunter Now](https://monsterhunternow.com/).
 
-If you aren't playing yet, [feel free to join](https://apps.apple.com/us/app/monster-hunter-now/id6445906110) and use my referral code for free items and zenny, `89K5MKCF`.
+In case you are new to Events, [Feel free to explore and join the enriching experience.](https://apps.apple.com/us/app/monster-hunter-now/id6445906110). Use the below referral code for free items and zenny, *`89K5MKCF`*.
 
-I created this because I didn't want to add events to a shard calendar for my friends and I any longer and it's also a great way to test out how beneficial AI is in coding via dialoge with ChatGPT, and an excuse to utilize the OpenAI API.
+This has been created to add events to a shared calendar for all the players including me and my friends. It's a great way to visualize how AI is beneficial for coding via dialogue with ChatGPT, and an exemplary way of utilizing the OpenAI API. 
 
-## Subscribe to Calendar
+### Steps to Subscribe the Event Calendar
 
 Add the [ICS file](https://seriouslysean.github.io/monster-hunter-now-events/dist/events.ics) to your calendar app.
 
-```
-https://seriouslysean.github.io/monster-hunter-now-events/dist/events.ics
-```
 
-## Run Locally
+https://seriouslysean.github.io/monster-hunter-now-events/dist/events.ics
```

Comment:
This url is in a code block to make it easier for users to copy it, so I'd like to retain that functionality.